AssetManagementTestProject/TestCases/US303LogoutTest.cs

```diff
@@ -0,0 +1,48 @@
+﻿using NUnit.Framework;
+using NUnit.Framework.Internal;
+using AssetManagementTestProject.PageObj;
+using AssetManagementTestProject.TestSetup;
+
+namespace AssetManagementTestProject.TestCase;
+
+[TestFixture]
+public class US303LogoutTest : NUnitWebTestSetup
```

Comment:
Unified naming. I think we should name each test script as `US(Num)_(Function)Test` e.g. US302_LoginTest

Reply:
You have yet to resolve this and other comments